src/connectdlgbase.ui

```diff
@@ -107,6 +107,9 @@
        </property>
       </widget>
      </item>
+     <item>
```

Comment:
The button looks a bit odd on macOS
![macOS connect dialog with backspace deletion button](https://github.com/jamulussoftware/jamulus/assets/20726856/3dbb22f2-4f7b-4d81-92ea-53a072e39b30), 

can you use another one, like the cancel button?

Reply:
I've looked through the Qt options and didn't find anything else.

Reply:
So you can't just use the same button?

Reply:
OK, now you've lost me.

Reply:
I meant that Qt creator should show the button (Cancel, Connect below the ip/server input field) in the gui. Can't you use the same button style for the button you added?

Reply:
Do you mean there would be three buttons? "Cancel", "Delete", "Connect"? I'd be slightly worried about fat-fingering the delete my mistake perhaps.

Reply:
Sorry for being unclear. But I meant that the button "Cancel" has some styling. This styling should be applied to the new button which is added here (QPushButton not QToolButton)

Reply:
OK, so that's what it looks like as a QToolButton
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/0ba8dc04-1502-4772-b7f9-8f6450bff110)
If I literally replace "QToolButton" with "QPushButton" in the `.ui` file, it looks like this:
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/480654ac-07c8-4aee-bd24-3dba6a6f8a6f)
(Don't ask me why it's that big, I've no idea...)

With a bit of a struggle I can get it looking like this in Qt Creator
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/ab80cfee-9692-44d3-9aca-b4395be5de5a)
but at run time, it still ends up looking like this
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/98607156-4327-4338-8d28-54bef25cc229)

In all cases, it's taller than the QPushButtons.

Reply:
Aha!
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/224130cc-bddf-42ac-b8c9-aac4049d94af)
That looks better.

Reply:
Will check on macOS.

Reply:
Seems as if nothing changed on macOS

Reply:
Can you post another screenshot, please?

Reply:
@ann0see if you get a chance to look at this, the screenshot would help me understand what you're seeing on MacOS.

Reply:
Sorry for the delay, but it looks basically the same:
![Jamulus delete button screenshot](https://github.com/jamulussoftware/jamulus/assets/20726856/bccf9e1f-c195-41f8-b962-1a4e6cefa552)

Reply:
Well, at least it's centred vertically...  I can try dropping the vertical max size.
```
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
```
`<sizepolicy hsizetype="Maximum" vsizetype="Ignored">` to `<sizepolicy hsizetype="Maximum" vsizetype="Maximum">` and `<property name="maximumSize">`->`<size>`->`<height>16777215</height>` to 24.

(I really just want to tell it "no smaller than the glyph, fill the height of the line that's already there".  But nothing I've read tells me how to do that.)